jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/JavaProject.java

```diff
@@ -271,7 +264,6 @@ public JavaProject(
     this.documentationUrl = documentationUrl;
     this.buildFinalName = buildFinalName;
     this.artifact = artifact;
-    this.localRepositoryBaseDirectory = localRepositoryBaseDirectory;
```

Comment:
You need also to remove the field from the builder.
Plus its setter in MavenUtil.